release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -95,6 +95,51 @@ You can now install a cluster on Google Cloud Platform (GCP) and use a personal
 
 //Link for more info when available.
 
+[id="ocp-4-7-osp-bare-metal-machines"]
+==== Installing a cluster on {rh-openstack} that uses bare metal machines
+
+You can now install a cluster on {rh-openstack-first} that uses bare metal machines. The cluster can have both control plane and compute machines running on bare metal, or just compute machines.
+
+// Link when available.
+
+[id="ocp-4-7-osp-validations"]
+==== Validating {rh-openstack} requirements at installation
+
+The {product-title} installer now performs a variety of checks before attempting to install a cluster on {rh-openstack}. The installer can identify certain conditions that cause installations to fail, such as overlapping subnets.
+
+[id="ocp-4-7-osp-custom-subnets"]
+==== Custom subnets for machines on {rh-openstack}
+
+You can now configure machines in clusters that run on {rh-openstack} to use a network and subnet of your choice.
```

Comment:
So, I believe we're talking about configure _new_ machines to run on a different subnet/network - there is still only one subnet possible for control plane and computes at install time. This should probably be clarified. 

And here again, the use case targets additional computes.

Reply:
Hopefully addressed in https://github.com/openshift/openshift-docs/pull/28504/commits/a72393ff5f366d706d23705e3a24ebd747b5759a